docs/typescript.md

```diff
@@ -201,15 +205,70 @@ Advanced Service currently do not have TypeScript autocompletion.
 
 ### TypeScript Support
 
-Currently, `clasp` supports [`typescript@2.9.2`](https://www.npmjs.com/package/typescript/v/2.9.2). If there is a feature in a newer  TypeScript version that you'd like to support, or some experimental flag you'd like enabled, please file a bug.
+Currently, `clasp` supports [`typescript@3.3.3`](https://www.npmjs.com/package/typescript/v/3.3.3). If there is a feature in a newer TypeScript version that you'd like to support, or some experimental flag you'd like enabled, please open an issue.
+
+### Modules, exports and imports
+
+Currently, Google Apps Script does not support modules, hence the typical `export`/`import` patern cannot be used and the example below will fail:
+
+```ts
+// module.ts
+export const foo = 'Hello from Module';  // `foo` is added to the `exports` object in the global namespace
+```
+
+```ts
+// main.ts
+import { foo } from './module';  // this statement is ignored
+
+const bar = foo;  // the variable `foo` does not exist in the global namespace
+```
+
+There are some possible workaround though:
+
+#### The `exports` declaration workaround
+
+The idea here is to make TypeScript aware of the `exports` object and its imported content.
```

Comment:
This workaround makes TypeScript aware of the `exports` object and its imported content.